dev_ops/sql/Tables/CreateFeaturedContracts.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE featured_contracts (
+    id BIGSERIAL PRIMARY KEY,
+	contract_id BIGINT REFERENCES contracts (id),
+	hype_text TEXT NOT NULL
```

Comment:
Same text comment here

Reply:
I think hype_text makes sense to leave as a TEXT field. The length of this text could change in the future so we shouldn't limit ourselves to picking a limit.

Reply:
Going to keep this one as TEXT.

Reply:
I still dont love TEXT. Leaving a field open to 60k characters is silly. we'll never need that much. 2048 is more than enough characters for what we'll use. Id prefer a limit but if you feel strongly it doesnt need one im fine  leaving it as is.